pywebio/platform/fastapi.py

```diff
@@ -151,8 +151,41 @@ def start_server(applications, port=0, host='',
 
     .. versionadded:: 1.3
     """
-    kwargs = locals()
+    app = build_starlette_app(allowed_origins, applications, cdn, check_origin, debug, static_dir)
+
+    if auto_open_webbrowser:
+        asyncio.get_event_loop().create_task(open_webbrowser_on_server_started('localhost', port))
+
+    if not host:
+        host = '0.0.0.0'
+
+    if port == 0:
+        port = get_free_port()
+    uvicorn.run(app, host=host, port=port)
+
+
+def build_starlette_app(applications, cdn=True, static_dir=None, debug=False, allowed_origins=None, check_origin=None):
```

Comment:
I think it's better to use `asig_app` as the function name:
```suggestion
def asig_app(applications, cdn=True, static_dir=None, debug=False, allowed_origins=None, check_origin=None):
```
References to `build_starlette_app()` elsewhere also need to be modified accordingly

Because I plan to add `wsgi_app()` or `asgi_app()` to the other backend modules as well, , so it can keep the naming uniformity